calcom/cal.com
# fix: Connect selected calendar to credential

## What does this PR do?



Stacked on #11281

Fixes # (issue)



## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change





## How should this be tested?



- Connect a calendar app
- Select calendars to check for availability
- In the DB the selected calendars should be connected to the credential 

## Mandatory Tasks


## Checklist




- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
